src/common/commonutils/SshUtils.c

```diff
@@ -528,6 +528,7 @@ int CheckSshProtocol(char** reason, void* log)
     else if (EEXIST == (status = CheckLineNotFoundOrCommentedOut(g_sshServerConfiguration, '#', protocol, log)))
     {
         OsConfigLogInfo(log, "CheckSshProtocol: '%s' is found uncommented in %s", protocol, g_sshServerConfiguration);
+        OsConfigCaptureSuccessReason(reason, "%s'%s' is found uncommented in %s", protocol, g_sshServerConfiguration);

```

Comment:
nit: Should the last %s be covered in single quotes for consistency with others?

Reply:
This is by design and yes, it's already consistent. This last string is the SSH server configuration file and simply it looks better there in reasons w/o the '' delimiters which are reserved for values.

Reply:
" 'Protocol 2' is found uncommented in /etc/ssh/sshd_config"

Same as for the sshd service here:

" The sshd service reports that 'clientaliveinterval' is set to '3600' (that is greater than zero)"

Reply:
sshd and /etc/ssh/sshd_config do not get quoted, they are the two main pieces there and are always the same

The values (which can change) get quoted and that only to see if there are spaces sneaking after and such